apps/docs/pages/guide/command-file-setup.mdx

```diff
@@ -290,15 +293,15 @@ This property contains the command's structural information, and is required to
 
 ### `run`
 
--   Type: `void`
+-   Type: `Function`
```

Comment:
this seems to be the return type of `run()`

Reply:
as I said, beginners (to javascript) can be confused with the type